src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -744,6 +769,18 @@ export default {
 	100% { background-color: rgba(var(--color-background-hover), 0); }
 }
 
+.new-message-marker {
+	margin: 20px 0;
+	text-align: center;
+
+	span {
+		background-color: var(--color-primary-light);
+		color: var(--color-text-lighter);
```

Comment:
I think this is too little contrast depending on the colors, cc @jancborchardt for the math
![Bildschirmfoto von 2021-03-01 11-05-28](https://user-images.githubusercontent.com/213943/109482270-1616e800-7a7e-11eb-9a09-ca29ce8b3baa.png)

Reply:
obsoleted by the previous request to have the bar look like the old dates:
<img width="501" alt="image" src="https://user-images.githubusercontent.com/277525/109503099-2046df80-7a9a-11eb-9779-8d1815c0d860.png">

Reply:
Well not requests but thoughts, don't know what's a good idea